tests/parm/rrfs_conus13km_hrrr.nml.IN

```diff
@@ -44,7 +41,7 @@
 
 &fms_nml
     clock_grain = 'ROUTINE'
-    domains_stack_size = 5000000 ! Smoke needs a higher value than 3000000 default.
```

Comment:
Disabling smoke is not an acceptable way to reduce the domains_stack_size required by smoke.